packages/config/config/devices/0x015f/mh-dt311_411.json

```diff
@@ -1,10 +1,10 @@
-// McoHome Technology Co., Ltd MH-DT311/MH-DT411
-// MCOHome Dimmer Switch
+// McoHome Technology Co., Ltd. MH-DT311 / MH-DT411 / MH-DT511
+// Dimmer Switch
 {
 	"manufacturer": "McoHome Technology Co., Ltd.",
 	"manufacturerId": "0x015f",
-	"label": "MH-DT311/MH-DT411",
-	"description": "MCOHome Dimmer Switch",
+	"label": "MH-DT311 / MH-DT411 / MH-DT511",
+	"description": "Dimmer Switch",
```

Comment:
Please carry all of my changes to this file through to other files.

Reply:
Will do.